app/javascript/react/components/RideShow.js

```diff
@@ -23,6 +24,12 @@ const RideShow = (props) => {
         <img src={props.image_url} alt={props.name} className="image-tile" />
       </div>
         {reviewTiles}
+
+        <ReviewForm 
+          reviews={props.reviews}
+          setReviews={props.setReviews}
+          addNewReview={props.addNewReview}
+        />
```

Comment:
`reviews` and `setReviews` aren't used in the form and should be removed as props